src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -39,6 +39,10 @@
 <H2>X-Content-Type-Options</H2>
 
 <H2>X-Frame-Option</H2>
+This scanner checks for the existence and validity of the X-FRAME-OPTIONS header. At HIGH threshold this scanner
+does not alert on client or server error responses, for all other enabled thresholds it alerts on all response 
+types if the header is missing or invalid. Invalid meaning that the header is present with no value, or that 
+the value is not as expected. 
```

Comment:
Maybe I'm asking too much ;)
It should say what are the expected values. I know that the user can see that in the solution, but it's another step and help pages should contain enough information to know what (and how) the scanner does [1].

[1] https://github.com/zaproxy/zaproxy/wiki/FAQzaptests